config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml

```diff
@@ -223,6 +223,7 @@ presubmits:
         - --test-cmd-args=--testoverrides=./testing/load/experimental/overrides/enable_jobs.yaml
         # TODO(https://github.com/kubernetes/perf-tests/issues/803): Re-enable
         # - --test-cmd-args=--testoverrides=./testing/load/experimental/overrides/enable_pvs.yaml
+        - --env=CL2_ENABLE_PVS=false # TODO(https://github.com/kubernetes/perf-tests/issues/803): Fix me
```

Comment:
Let's move the --env up (as you had in the other PR). Here we put the CL2 args and it's not a CL2 arg (at lest not directly).

Reply:
Thanks. Moved the line up to keep change consistent with https://github.com/kubernetes/test-infra/pull/16849